notebooks/worldcereal_v1_demo_custom_cropland.ipynb

```diff
@@ -11,90 +11,56 @@
-    "\tmaxLevel=6\n",
-    "\t/vscode-jupyter-toc-config -->\n",
-    "<!-- THIS CELL WILL BE REPLACED ON TOC UPDATE. DO NOT WRITE YOUR TEXT IN THIS CELL -->"
+    "### Introduction\n",
+    "\n",
+    "This notebook guides you through the process of training a custom cropland classification model using publicly available and harmonized in-situ reference data for your area of interest. Afterwards, the model can be applied to your area and season of interest to generate a cropland extent map.\n",
+    "\n",
+    "Please note that for the purpose of this demo, the processing area is currently limited to 250 km² per model run."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# <a id='toc1_'></a>[Before you start](#toc0_)\n",
-    "\n",
-    "In order to run WorldCereal crop mapping jobs from this notebook, you need to create an account on the Copernicus Data Space Ecosystem (CDSE) registering [here](https://dataspace.copernicus.eu/). This is free of charge and will grant you a number of free openEO processing credits to continue this demo."
+    "### Content\n",
+    "  \n",
+    "- [Before you start](###-Before-you-start)\n",
+    "- [1. Define your region of interest](#1.-Define-your-region-of-interest)\n",
+    "- [2. Extract public reference data](#2.-Extract-public-reference-data)\n",
+    "- [3. Create your custom cropland class](#3.-Create-your-custom-cropland-class)\n",
+    "- [4. Prepare training features](#4.-Prepare-training-features)\n",
+    "- [5. Train custom classification model](#5.-Train-custom-classification-model)\n",
+    "- [6. Deploy your custom model](#6.-Deploy-your-custom-model)\n",
+    "- [7. Generate a map](#7.-Generate-a-map)\n"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 1,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# TEMPORARY CELL\n",
+    "### Before you start\n",
     "\n",
-    "import sys\n",
-    "sys.path.append('/home/jeroendegerickx/git/worldcereal/worldcereal-classification/notebooks')\n",
-    "%load_ext autoreload\n",
-    "%autoreload 2"
+    "In order to run WorldCereal crop mapping jobs from this notebook, you need to create an account on the Copernicus Data Space Ecosystem (CDSE) registering [here](https://dataspace.copernicus.eu/). This is free of charge and will grant you a number of free openEO processing credits to continue this demo."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# <a id='toc2_'></a>[Define a region of interest](#toc0_)\n",
+    "### 1. Define your region of interest\n",
     "\n",
     "When running the code snippet below, an interactive map will be visualized.\n",
     "Click the Rectangle button on the left hand side of the map to start drawing your region of interest.\n",
+    "\n",
     "Currently, there is a maximum size of 250 km² for your area within this demo. Upon exceeding this limit, an error will be shown.\n",
```

Comment:
I think we should tackle it as follows:
- have a default max of 250km²
- if between 250km² and 750km² -> show the error but provide users with info how to manually raise the limit to max 750 km², pointing them to the credit cosrt
- if >750km², consider this as a hard maximum

Reply:
Currently, there is a maximum size of 250 km² for your area within this demo. Upon exceeding this limit, an error will be shown.
You can bypass this limit by altering the code below to:<br>
*map = ui_map(area_limit=750)*<br>

Processing areas beyond 750 km² are currently not supported.

Reply:
don't forget to mention the actual reason for this: credit usage :-) otherwise a user may think it's odd why this limit is there in the first place